### DISCUSSION TITLE
A new SOTA text recognition architecture - SVIPTR

### DISCUSSION THREAD

**Discussion by milosacimovic** [mindee/doctr#1826] [Ideas]
Hi, 

I would like to suggest possibly introducing another state-of-the-art text recognition architecture to docTR.
[SVIPTR](https://paperswithcode.com/paper/viptr-a-vision-permutable-extractor-for-fast)
It's promising accurate results at low latency.

Notably, the SVIPTR-T (Tiny) variant delivers highly competitive accuracy on par with other lightweight models and achieves SOTA inference speeds. Meanwhile, the SVIPTR-L (Large) attains SOTA accuracy in single-encoder-type models, while maintaining a low parameter count and favorable inference speed.

Thanks for your consideration.

**Comment by felixdittrich92** [comment]
Hi @milosacimovic 👋🏼,

Thanks for sharing with us, I will have a look on it after vacation 😊
Does not look like there are benchmarks against parseq / vitstr actually right ?

**Comment by milosacimovic** [comment]
Hi @felixdittrich92 👋,

Happy holidays! 🎄
Here's their [benchmark table](https://github.com/cxfyxl/viptr#results-on-benchmark-datasets-and-comparison-with-sota)
It does not include parseq but it seems it's a bit less accurate on IC13 dataset. However, the large variant seems to be better on average than some really big models like TrOCR for example while being a lot faster and smaller.

**Reply by felixdittrich92** [reply]
Thanks wish you too a few days to relax 👍🏼🤗
I quickly checked the paper and benchmarks today and it looks like a valid candidate to be added to docTR .. so feel free to open an issue ticket.. from inference latency it should be comparable with `crnn_mobilenet_v3_large` by keeping the capacity comparable to `parseq`☺️

**Reply by felixdittrich92** [reply]
Would you like to work on it ? :)

**Comment by felixdittrich92** [comment]
#1867
